extensions/sql-migration/src/dialog/skuRecommendationResults/skuRecommendationResultsDialog.ts

```diff
@@ -365,6 +378,25 @@ export class SkuRecommendationResultsDialog {
 		}
 	}
 
+	private getDiskTypeText(type: contracts.AzureManagedDiskType): string {
```

Comment:
I suggest creating a lookup table pattern in constants.ts for this, similar to the ParallelCopyType const

export const ParallelCopyType: LookupTable<string | undefined> = {
	[ParallelCopyTypeCodes.None]: localize('sql.migration.parallel.copy.type.none', 'None'),
	[ParallelCopyTypeCodes.DynamicRange]: localize('sql.migration.parallel.copy.type.dynamic', 'Dynamic range'),
};

Reply:
Done.